src/main/java/com/carrotcreative/cream/cache/CacheManager.java

```diff
@@ -33,6 +34,8 @@ public static CacheManager getInstance(Context context){
 
     private final File mRootDir;
     private final Context mContext;
+    private boolean mLruCacheEnabled;
```

Comment:
I know this defaults to false, but could we just say `private boolean mLruCacheEnabled = false;` for clarity :D?